docs/developers/meta/sc-meta-cli.md

```diff
@@ -114,6 +114,9 @@ Paramameters:
 - `--to`
     - Overrides the version to upgrade to.
     - _default_: the last released version.
+- `--no-check`
+    - Skips `cargo check` after upgrade.
+    - _default_: project is compiled.
```

Comment:
A few more words here:
By default `upgrade` compile checks the project after each major version upgrade. This is to allow developers that upgrade multiple versions to address issues with the upgrade before too many such issues get to accumulate.
This feature can be turned off by the --no-check flag.